test/e2e/atlas/helper_test.go

```diff
@@ -699,7 +699,7 @@ func deleteAllClustersForProject(t *testing.T, cliPath, projectID string) {
 					_ = watchCluster(projectID, clusterName)
 					return
 				}
-				assert.NoError(t, deleteClusterForProject(projectID, clusterName))
+				require.NoError(t, deleteClusterForProject(projectID, clusterName))
```

Comment:
same as above, we want to try all

Reply:
removed in [0d3326d](https://github.com/mongodb/mongodb-atlas-cli/pull/2600/commits/0d3326d98c808cbe9b3d3497d5e6176079523b5a)